gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java

```diff
@@ -283,6 +285,11 @@ String mtlsEndpointResolver(
     }
 
     public EndpointContext build() throws IOException {
+      // If the universe domain wasn't configured explicitly in the settings, check the
```

Comment:
Can we move the logic to `determineUniverseDomain()`?

Reply:
Sure, moved it to `determineUniverseDomain()`